lib/order.rb

```diff
@@ -0,0 +1,104 @@
+require_relative 'Menu'
+require_relative 'Texter'
+class Order
+  attr_reader :pricelist
+
+  def initialize(set_menu: Menu)
+    @pricelist = set_menu.new
+  end
+  def place_order
+    intro
+    notepad = take_order
+    check_total(notepad)? gucci : miscalulated
+  end
+
+  protected
+
+  def take_order(dishes: 0, pad: [])
+    phone_order = gets.chomp
+    while !phone_order.empty? do
+      request = phone_order.split(', ')
+      volume = request[1].to_i
+      pad.push({request[0] => volume})
+      dishes += volume
+      puts "So that's #{dishes} plates so far"
+      phone_order = gets.chomp
+    end
+    pad
+  end
```

Comment:
I like the use of gets chomp and parsing a string, but what if they say "1, carbonara" instead of "carbonara, 1"?